packages/@react-aria/combobox/docs/useComboBox.mdx

```diff
@@ -303,7 +302,9 @@ function ListBox(props) {
       style={{
         margin: 0,
         padding: 0,
-        listStyle: "none"
+        listStyle: "none",
+        maxHeight: "150px",
+        overflow: "auto"
```

Comment:
the async example in useComboBox docs will now scroll, also is a good thing to add to the example since people will most likely want to limit the listbox height

Reply:
Can you add this to the `useSelect` example as well? The listbox there should be identical I believe.